cmd/crdCreator/main.go

```diff
@@ -41,6 +41,7 @@ func main() {
 		"noobaas.noobaa.io.yaml",
 		"objectbucketclaims.objectbucket.io.yaml",
 		"objectbuckets.objectbucket.io.yaml",
+		"storageclients.ocs.openshift.io.yaml",
```

Comment:
iirc, these are required for registering with scheme even before offering CR is created.

Reply:
We need them because we are trying to watch these resources and definitions for these resources are not available in the cluster at the time of registering watches.

Reply:
But we had a fix for that. What happened? 
The issue is that by introducing CRD ourselves, OLM will not override them when installing the offering package. This was only supposed to be used for CRDs that the offerings watch but NOT install.

Reply:
> But we had a fix for that. What happened?
Are you talking about adding dynamic watches?

Reply:
no, about the op restart solution

Reply:
We didn't do that and decided to install CRDs on our own using the init container.

Reply:
@bindrad I will approve this for this PR. But please find someone to work on this ASAP. 
We cannot ship with CRs that we are placing instead of the offering themselves. And the alternative will need time for testing